common/src/main/java/tachyon/Constants.java

```diff
@@ -261,4 +261,8 @@
   public static final int BYTES_WRITTEN_LOCAL_INDEX = 8;
   public static final int BYTES_WRITTEN_REMOTE_INDEX = 9;
   public static final int BYTES_WRITTEN_UFS_INDEX = 10;
+
+  private Constants() {
```

Comment:
``private Constants() {} // prevent instantiation`` is fewer lines of code and more informative